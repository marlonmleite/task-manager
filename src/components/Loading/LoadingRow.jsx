import React from 'react'
import { Loading, LoadingFullscreenStyled } from './styled'

const LoadingRow = () => (
  <LoadingFullscreenStyled background="rgba(255, 255, 255, 0.85)">
    <Loading size="20px" />
  </LoadingFullscreenStyled>
)

export default LoadingRow
