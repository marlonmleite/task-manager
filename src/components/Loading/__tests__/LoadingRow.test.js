import React from 'react'
import { shallow } from 'enzyme'
import LoadingRow from '../LoadingRow'

describe('LoadingRow', () => {
  it('should render', () => {
    const loading = shallow(<LoadingRow />)

    expect(loading).toMatchSnapshot()
  })
})
