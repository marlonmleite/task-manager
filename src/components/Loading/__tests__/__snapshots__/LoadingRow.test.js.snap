// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingRow should render 1`] = `
<styled.div
  background="rgba(255, 255, 255, 0.85)"
>
  <styled.div
    color="primary"
    size="20px"
  />
</styled.div>
`;
